frame/lending/src/lib.rs

```diff
@@ -1097,6 +1086,34 @@ pub mod pallet {
 
 			Ok(())
 		}
+
+		/// Check is price actual yet
+		fn ensure_price(market: &MarketConfigOf<T>) -> Result<(), DispatchError> {
```

Comment:
I would change this name to ensure_price_is_recent or something a little more verbose